src/datapump/sync/sync.py

```diff
@@ -974,6 +975,126 @@ def build_jobs(self, config: DatapumpConfig) -> List[Job]:
         return jobs
 
 
+class IntDistAlertsSync(Sync):
+    """
+    Defines jobs to create merged integrated-DIST alerts assets once integrated alerts raster is available.
+    """
+
+    DATASET_NAME = "gfw_integrated_dist_alerts"
+    SOURCE_DATASETS = [
+        # Integrated alerts must be first
```

Comment:
Could you say why please? I know because we just discussed the solo tiles implementation, but I think it would be good to persist here. :)